### ISSUE TITLE
Accessibility (major): Landing Page - Image alt text Stanford Libraries [prea-000]

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/pre-assembly#1255]
## Description

The Stanford Libraries image link should have appropriate alt text.

## Suggestion

```
<a href="/">
  <img height="30" src="/assets/sul-logo-73b22d6e1be87f315ab1f323142c2e94df42f07b.png" alt="Stanford Libraries home">
</a>
```

https://docs.google.com/spreadsheets/d/105GozrKuWrnXaEk5FoP4UbzHSCTPgeEqPaPiHi1c950/edit#gid=0
